docs/2.3/user-manual.md

```diff
@@ -113,6 +114,28 @@ total 8.0K
 -rw------- 1 joe staff 1.5K Aug 10 16:16 joe-cert.pub
 ```
 
+### SSH Certificates for Automation
+
+Regular users of Teleport must request an auto-expiring SSH certificate,
+usually every day. This doesn't work for non-intractive scripts, like cron jobs
+or CI/CD pipeline.
+
+It is recommended to create a separate Teleport user for such bots and request
+a certificate for them with a long time to live (TTL). In this example we're
+creating a certificate with a TTL of 10 years for the Jenkins user and storing
+it in jenkins.pem file, which can be later used with `-i` (identity) flag for `tsh`.
+
+```bash
+# to be executed on a Teleport auth server
+$ tctl auth sign --ttl=87600h --user=ekontsevoy --out=jenkins.pem
+```
+
+Copy jenkins.pem to the jenkins server and it can be used with `-i` (identity
```

Comment:
I think you suggested to stick to neutral style in docs vs referring directly with `you` or giving instructions? E.g. this should be: `The file jenkins.pem can be used on jenkins server...`